### ISSUE TITLE
Public IP on ALB?

### ISSUE THREAD

**Issue by Melbit-haixingzhang** [nathanpeck/aws-cloudformation-fargate#5]
Following pictures shows a static Public IP assigned to ALB which is same to NAT, possible mistaken IP?
https://github.com/nathanpeck/aws-cloudformation-fargate/raw/master/images/private-task-public-loadbalancer.png

**Comment by nathanpeck** [comment]
Yep that’s a mistake in the diagram. Will fix it if I get a chance
